cmd/rpcdaemon/cli/config.go

```diff
@@ -434,14 +434,19 @@ func RemoteServices(ctx context.Context, cfg httpcfg.HttpCfg, logger log.Logger,
 }
 
 func StartRpcServer(ctx context.Context, cfg httpcfg.HttpCfg, rpcAPI []rpc.API) error {
```

Comment:
```suggestion
func StartRpcServer(ctx context.Context, cfg httpcfg.HttpCfg, rpcAPI []rpc.API, authAPI []rpc.API) error {
```

Reply:
and you need to change the code in eth/backend.go to provide `authAPI`.

just with 2 entries, `engine API` and `eth API`, you can see how they are created in `https://github.com/ledgerwatch/erigon/blob/8e3ac8a21c8f55bdd78338e88e956999ad7e60b8/cmd/rpcdaemon/commands/daemon.go` line 15.

Reply:
Okay, should the authAPI also be predefined with two values in the case of `rpcdeamon` and `main`?

Reply:
yup

Reply:
Done! Please check again